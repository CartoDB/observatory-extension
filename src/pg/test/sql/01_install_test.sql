-- Install dependencies
CREATE EXTENSION postgis;
CREATE EXTENSION postgres_fdw;

-- Install the extension
CREATE EXTENSION observatory VERSION 'dev';
