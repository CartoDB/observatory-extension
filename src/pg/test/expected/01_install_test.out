-- Install dependencies
CREATE EXTENSION postgis;
CREATE EXTENSION plpythonu;
CREATE EXTENSION cartodb;
-- Install the extension
CREATE EXTENSION observatory VERSION 'dev';
